noxfile.py

```diff
@@ -146,9 +146,15 @@ def codeqa(session: nox.Session):
 
 @nox.session
 def typing(session: nox.Session):
-    install(session, ".[typing]", *other_antsibull(), editable=True)
+    others = other_antsibull()
+    install(session, ".[typing]", *others, editable=True)
     session.run("mypy", "src/antsibull")
 
+    additional_libraries = []
+    for path in others:
+        if isinstance(path, Path):
+            additional_libraries.extend(('--search-path', str(path / 'src')))
```

Comment:
The file currently uses double quotes ;)

```suggestion
            additional_libraries.extend(('--search-path', str(path / "src")))
```

Reply:
Oops!